apps/imports/filters.py

```diff
@@ -0,0 +1,37 @@
+"""
+Copyright 2019 ShipChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from functools import partial
+
+from django_filters import CharFilter
+from django_filters import rest_framework as filters
+
+from apps.utils import generic_filter_enum
+from .models import ShipmentImport
+
+
+# pylint:disable=invalid-name
+filter_enum = partial(generic_filter_enum, module_ref='apps.imports.models')
```

Comment:
I appreciate the efforts here to keep things DRY - but in this case I think that this partial is going to cause more of a headache than duplicate code. I think the filter_enum function can be copied to each app's filters.py